scripts/make-helpers.sh

```
@@ -17,27 +19,133 @@ hint() {
+  local elapsed=$(($t1-$t0))
   kv Exit "$rc"
-  kv Time "$(($t1-$t0))s"
+  kv Time "${elapsed}s"
+
+  # Consolidate stdout/stderr into a single combined file
+  local max_bytes=${D3K_LOG_MAX_CAPTURE:-65536}
+  local stdout_len=0 stderr_len=0
+  [[ -f "$out_file" ]] && stdout_len=$(wc -c <"$out_file" 2>/dev/null || echo 0)
+  [[ -f "$err_file" ]] && stderr_len=$(wc -c <"$err_file" 2>/dev/null || echo 0)
+  local stdout_trunc=false stderr_trunc=false
+  if (( stdout_len > max_bytes )); then stdout_trunc=true; fi
```

Shellcheck (0.11.0)

[warning] stdout_trunc appears unused. Verify use (or export if used externally).

(SC2034)